tls/s2n_resume.c

```diff
@@ -710,3 +710,22 @@ int s2n_config_store_ticket_key(struct s2n_config *config, struct s2n_ticket_key
     GUARD_AS_POSIX(s2n_set_add(config->ticket_keys, key));
     return S2N_SUCCESS;
 }
+
+int s2n_config_set_requested_new_ticket_count(struct s2n_config *config, uint8_t num)
+{
+    notnull_check(config);
+
+    config->initial_tickets_to_send = num;
```

Comment:
Is there an upper limit for this value? Do we want to enforce one here?

Reply:
The upper limit here is 256 tickets, which I think is a good upper limit for now.

Reply:
We could add a check here to verify this upper limit of tickets ( <= 256)